simulation_generator/slim/Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM ubuntu
+
+RUN apt-get update -y
+# install packages
+RUN apt-get install -y \
+    wget make cmake gcc g++ unzip
+
+# retrieve and install slim
+RUN wget http://benhaller.com/slim/SLiM.zip
+RUN unzip SLiM.zip \
+    && mkdir /build 
+WORKDIR /build
+RUN cmake /SLiM \
+    && make slim
+
+# create symlink to slim
+RUN ln -s /build/slim /usr/bin/slim
+WORKDIR /
+
+RUN apt-get install -y python-dev
```

Comment:
I think we should be explicit about which version of python we are downloading here so that when the image is built by others in the future it doesn't just reach for the current `dev`  version of python, which may or may not be the `dev` version we were working with when developing.